marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 2191273978937394,
+            "layout": {
+                "height": 27,
+                "width": 22,
+                "x": 0,
+                "y": 14
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "sum:marklogic.hosts.total_rate{$cluster} by {host}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Sum of the processing rate factors over $cluster",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 5983605072600952,
+            "layout": {
+                "height": 15,
+                "width": 47,
+                "x": 24,
+                "y": 65
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_blue",
+                "content": "Database summary",
+                "font_size": "14",
+                "show_tick": false,
+                "text_align": "center",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 3172903413969796,
+            "layout": {
+                "height": 5,
+                "width": 38,
+                "x": 128,
+                "y": 0
+            }
+        },
+        {
+            "definition": {
+                "background_color": "blue",
+                "content": "Requests",
+                "font_size": "14",
+                "show_tick": false,
+                "text_align": "center",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 6184667112701002,
+            "layout": {
+                "height": 5,
+                "width": 38,
+                "x": 128,
+                "y": 47
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "sum:marklogic.transactions.total_transactions{*}",
```

Comment:
Are there any tags we could use for grouping here? Template variables for filtering?

Reply:
Filter by `$cluster` (== `host`)